### PR TITLE
Unsteady features for explicit time stepping

### PR DESCRIPTION
**Pull request by demanosalvas**
Adds improvements for the use of explicit time stepping for unsteady problems

### CODE REVIEW

File: SU2_CFD/src/solver_direct_mean.cpp [hunk 1]
```diff
@@ -6412,6 +6432,11 @@ void CEulerSolver::BC_Euler_Wall(CGeometry *geometry, CSolver **solver_container
         VelMagnitude2_i = 0.0; ProjVelocity_i = 0.0;
         for (iDim = 0; iDim < nDim; iDim++) {
           Velocity_i[iDim] = node[iPoint]->GetVelocity(iDim);
+          
+          Xcoord = geometry->node[iPoint]->GetCoord(0);
+          if (Xcoord < 0.01 || Xcoord > 0.99){
```
Comment (talbring) [inline SU2_CFD/src/solver_direct_mean.cpp]: This could be removed. I don't think that this is intended.

**Comment by talbring** [comment]
The changes in SU2_CFD.cpp look a little bit strange, because git thinks that the whole file has changed (which is not true I guess).

**Comment by economon** [comment]
Looks like there are some conflicts in this one... @demanosalvas, perhaps we can catch up about this in person on Thursday to make sure there are no roadblocks?

File: SU2_CFD/src/solver_direct_mean.cpp (hunk 1 above)
Comment (demanosalvas) [inline SU2_CFD/src/solver_direct_mean.cpp]: totally agree, I missed it when I was removing shock tube stuff

**Pull request closed without merging**